src/VolleyManagement.Backend/VolleyManagement.Services/TeamService.cs

```diff
@@ -123,10 +123,24 @@ public void AddPlayers(TeamId team, IEnumerable<PlayerId> players)
                 throw new MissingEntityException(ServiceResources.ExceptionMessages.TeamNotFound, team.Id);
             }
 
+            CheckIfPlayersAreNotInAnoutherTeam(players);
+
             changedTeam.AddPlayers(players);
             _teamRepository.Update(changedTeam);
         }
 
+        private void CheckIfPlayersAreNotInAnoutherTeam(IEnumerable<PlayerId> addedPlayers)
+        {
+            var playersTeams = addedPlayers
+                .Select(x => _getPlayerTeamQuery.Execute(new FindByPlayerCriteria { Id = x.Id }));
+            //check if players play in another team
+            if (playersTeams.Any(x => x > 0))
```

Comment:
what does it mean x>0?

Reply:
It checks if any player is a member of any team

Reply:
if no 'x' will be equals to null

Reply:
_getPlayerTeamQuery returns id of team in form of **int**, in which players is a member. If there is no such a team -  '0' is returned.